x-pack/plugins/maps/index.js

```diff
@@ -79,6 +79,33 @@ export function maps(kibana) {
       const xpackMainPlugin = server.plugins.xpack_main;
       let routesInitialized = false;
 
+      xpackMainPlugin.registerFeature({
+        id: 'maps',
+        name: i18n.translate('xpack.maps.featureRegistry.mapsFeatureName', {
+          defaultMessage: 'Maps',
+        }),
+        icon: 'gisApp',
+        navLinkId: 'maps',
+        app: ['maps', 'kibana'],
```

Comment:
can you reuse constant declaration? https://github.com/thomasneirynck/kibana/blob/7c297cd60ad7766dffb6ec6c02cfe80f38b9df90/x-pack/plugins/maps/common/constants.js#L24

Reply:
Sure thing!